check/doctest

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Runs python doctest on all python source files in the cirq directory.
+#
+# See also:
+#     https://docs.python.org/3/library/doctest.html
+#
+# Usage:
+#     check/doctest [-q]
+#
+# The -q argument suppresses all output except the final result line and any
+# error messages.
+################################################################################
+
+# Get the working directory to the repo root.
+cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$(git rev-parse --show-toplevel)"
+
+PYTHONPATH="$(pwd)" python dev_tools/run_doctest.py $@
```

Comment:
nit: it's safer with quotes: `"$@"` (note that bash is smart when quotes are used with `$@` - the result is equivalent to quoting each element in `$@`, not to coalescing all array elements into one quotes string - which wouldn't do what we need)

Reply:
Done